src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -86,7 +87,12 @@ class WorkspaceInvitePage extends React.Component {
 
     render() {
         return (
-            <ScreenWrapper>
+            <ScreenWrapper onTransitionEnd={() => {
+                if (this.emailOrPhoneInputRef) {
+                    this.emailOrPhoneInputRef.focus();
+                }
```

Comment:
Can we make this use an early return please to match our other JS styles?

```js
if (!this.emailOrPhoneInputRef) {
    return;
}

this.emailOrPhoneInputRef.focus();
```

also it wouldn't hurt to make this a method on the class and just reference `this.focusEmailOrPhoneInput`.

Reply:
> Can we make this use an early return please to match our other JS styles?

That's fine by me. 
But I think we have used this type of code almost everywhere :

https://github.com/Expensify/App/blob/2f9aa32b0a9b49d720c81657048b7e0519db2b08/src/pages/home/report/ReportActionCompose.js#L398-L402

https://github.com/Expensify/App/blob/2f9aa32b0a9b49d720c81657048b7e0519db2b08/src/pages/iou/steps/IOUAmountPage.js#L85-L91

https://github.com/Expensify/App/blob/2f9aa32b0a9b49d720c81657048b7e0519db2b08/src/pages/settings/AddSecondaryLoginPage.js#L118-L123

https://github.com/Expensify/App/blob/2f9aa32b0a9b49d720c81657048b7e0519db2b08/src/pages/settings/PasswordPage.js#L109-L114

https://github.com/Expensify/App/blob/2f9aa32b0a9b49d720c81657048b7e0519db2b08/src/pages/settings/Payments/AddPayPalMePage.js#L68-L73

> also it wouldn't hurt to make this a method on the class and just reference this.focusEmailOrPhoneInput.

Sure :+1:

Reply:
Will wait for the reply before making any changes. Fixed the merge conflicts for now.

Reply:
Hah well, I can only enforce the rules that I catch people violate :) We prefer using early returns please update just your code only to follow the style rules thanks.

Reply:
Haha, Sure. I just wanted the green light before updating the code.